### DISCUSSION TITLE
Chess.com with DGT Centaur

### DISCUSSION THREAD

**Discussion by EdNekebno** [EdNekebno/DGTCentaurMods#79] [Show and tell]
Emulation is good enough now that I can link into the Android chess.com app to play on chess.com on the DGT Centaur! 

https://twitter.com/ednekebno/status/1458073981706280965

Still a bit buggy...but close.

**Comment by SnowBikeVR013** [comment]
hello,very Nice !!  I can't wait to test ! when you will give us an update so that we can play with it?

**Reply by SnowBikeVR013** [reply]
Hi, have you news with DGT on chess.com ? Works now ? An update perhaps ?.. ;-)

**Comment by wormstein** [comment]
Hi. No updates on this and doubt it will happen soon as chess.com won't be happy about this direct integration resulting in user's accounts being blocked once used with DGTCentaurMods. How about using the eBoard emultaion with this? It should work.

**Reply by SnowBikeVR013** [reply]
are you talking about the dgt chess application with android?  or whitepawn?
